src/content/1.7/modules/concepts/templating/vuejs/faq.md

```diff
@@ -27,6 +27,77 @@ Example:
     ]);
 ```
 
+### How to send data from Symfony modern controllers to Vue.js?
+
+PHP controller:
+
+```php
+namespace Moduel\YorModule\Controller\Admin;
+
+use PrestaShopBundle\Controller\Admin\FrameworkBundleAdminController;
+
+class YourControllerNameController extends FrameworkBundleAdminController
+{
+    public function demoAction()
+    {
+        return $this->render('@Modules/your-module/views/templates/admin/demo.html.twig', [
+            'yourModule' => [
+                'keyA' => 'valueA',
+                'keyB' => 'valueB',
+            ]
+        ]);
+    }
+}
+```
+
+Twig template:
+
+```twig
+{% extends '@PrestaShop/Admin/layout.html.twig' %}
+{% trans_default_domain "Module.YourModule.Admin.YourControllerName" %}
+
+{% block stylesheets %}
+  <link rel="stylesheet" href="https://ui-kit.prestashop.com/backoffice/latest/css/bootstrap-prestashop-ui-kit.css" type="text/css" media="all">
+  <link rel="stylesheet" href="{{ asset('../modules/your-module/views/css/app.css') }}" type="text/css" media="all">
+{% endblock %}
+
+{% block content %}
+  <div id="app"></div>
+{% endblock %}
+
+{% block javascripts %}
+    {{ parent() }}
+
+    <script>
+      var yourModule = {{ yourModule|json_encode|raw }}
+    </script>
+
+    <script src="https://ui-kit.prestashop.com/backoffice/latest/js/prestashop-ui-kit.js"></script>
```

Comment:
same, I'd go for local path, we must have ui-kit globally with PrestaShop, right @NeOMakinG ?

Reply:
Even if it's an example with external paths, it could lead to misunderstanding, as the UIKit is already included inside PS, maybe we could use another example? Maybe a dummy domain

Reply:
Sure. Let me change that.

Reply:
Done. I modified and included comments on Twig to clarify asset examples.